projects/10up-theme/theme-config/settings.jsonc

```diff
@@ -0,0 +1,43 @@
+{
+  // Define $schema so that autocomplete and inline validation works in supported IDEs.
+  "$schema": "https://schemas.wp.org/wp/6.0/theme.json",
```

Comment:
@christianc1 

I think this is duped from base.jsonc - only need to declare schema once

Reply:
You're right, it's duplicated in both files, but IDE's that support $schema validation use that attribute to provide inline autocomplete, validation, and typehinting.  So it's defined in all the files so you can take advantage of that functionality regardless of what file you're editing.  The compiled version only includes it once.